### ISSUE TITLE
simple.elm example does not compile

### ISSUE THREAD

**Issue by martinos** [etaque/elm-form#20]
The Simple.elm example app does not compile:

```
Detected errors in 1 module.
-- NAMING ERROR ------------------------------------------------- src/Simple.elm

Cannot find variable `:=`

42│     ("bar" := email)
         ^^^^^^^^^^^^^^
Maybe you want one of the following?

    /=
    ::
    <=
    ==

-- NAMING ERROR ------------------------------------------------- src/Simple.elm

Cannot find variable `:=`

43│     ("baz" := bool)
         ^^^^^^^^^^^^^
Maybe you want one of the following?

    /=
    ::
    <=
    ==

-- NAMING ERROR ------------------------------------------------- src/Simple.elm

Cannot find variable `Form.submit`.

92│           [ onClick formAddress Form.submit ]
                                    ^^^^^^^^^^^
`Form` does not expose `submit`. Maybe you want one of the following?

    Form.Submit

```

**Comment by etaque** [comment]
Ah, good find, forgot to update this one. So you found the action error, for the `:=` it's because I've removed infix operators, too much conflicts & bugs with other packages. Should be written like that now: `get "bar" email`. Could you add this to you PR? thanks :)

**Comment by etaque** [comment]
Fixed with 3.0.0
